docs/05-PipelineLaunchOptions.md

```diff
@@ -105,6 +105,20 @@ Example for two entries/scans to be uploaded (one human subject and one phantom)
 
 ## 5.2 - Pipeline Launch for BIDS datasets
 
+### 5.2.1 - Ideal BIDS structure for LORIS
+
+The BIDS dataset to import has to:
+
+- pass the BIDS validator of [pybids](https://github.com/bids-standard/pybids)
+- contain a `participants.tsv` file at its root. Ideally, this file will contain the following columns:
+  - `participant_id`: the PSCID of the candidate  
+  - `date_of_birth`: the date of birth (in YYYY-MM-DD) to use when creating the candidate in LORIS 
+  - `sex`: the sex to use when creating the candidate in LORIS
+  - `site`: the name of the LORIS site to associate the candidate with when creating the candidate in LORIS
+  - `project`: the name of the LORIS project to associate the candidate with when creating the candidate in LORIS
```

Comment:
This field is optional. If provided, the entry should match the name of a project already populate in the database. If missing or invalid, NULL will be used instead.

Reply:
Maybe another comment can be added to indicate that a subproject can also be specified (see https://github.com/aces/Loris-MRI/pull/558)

Reply:
Lastly, it can be great to specify that in case one of site/project/subproject is missing, NULL is inserted in the session table and later manual addition is required in order to display the entry on the frontend.

Reply:
thanks! All updated in the last commit

Reply:
Awesome. Seems perfect.